app/controllers/requests/received_requests_controller.rb

```diff
@@ -1,4 +1,8 @@
 class Requests::ReceivedRequestsController < ApplicationController
   def index
+    receivers = Receiver.where(user: current_user)
+    @requests = []
+    receivers.each { |receiver| @requests << receiver.request }
+    @requests
```

Comment:
Yes, I think the last choice (creating new association) is the best.  I'll make this association.